### PR TITLE
fix(build): Remove babel requires and unneeded babel traversal step

### PR DESCRIPTION
**Pull request by ewanharris**
Fixes https://jira.appcelerator.org/browse/TIMOB-26564. 

This code is required for hyperloop 2.x which is not supported by SDK 8

This should fix the current build issues on Jenkins fyi @ssjsamir

### CODE REVIEW

File: cli/commands/_build/copy.js [hunk 1]
```diff
@@ -717,7 +649,7 @@ function copyResources(next) {
 								}
 							})(r, from, to, cb);
 						})(from, to, cb);
-					})(from, to, ast, traverse, types, done);
+					})(from, to, done);
```
Comment (ewanharris) [inline cli/commands/_build/copy.js]: I think these have to stay to be compatible with 3.x versions of hyperloop. Need to investigate

Comment (ewanharris) [reply cli/commands/_build/copy.js]: But if they stay that means we need to bring back babel-*, erugh

**Comment by sgtcoolguy** [comment]
I assume this then requires appcelerator/hyperloop.next#310 to be merged too (and a new hyper loop to be released with 8.0.0)?

**Comment by ewanharris** [comment]
@sgtcoolguy yup, I'm guessing we'll maybe do hyperloop 3.2.0 or hyperloop 4.0.0 with SDK 8.0.0

File: cli/commands/_build/copy.js (hunk 1 above)
Comment (infosia) [reply cli/commands/_build/copy.js]: I think we can remove 3.x compatibility for upcoming 8.0.0 sdk. we should be able to release new version.

**Pull request merged**